gdal/port/makefile.vc

```diff
@@ -107,13 +107,29 @@ EXTRAFLAGS =	$(EXTRAFLAGS) -DHAVE_OPENSSL_CRYPTO $(OPENSSL_INC)
 ODBC_OBJ =	cpl_odbc.obj
 !ENDIF
 
-default:	cpl_config.h $(LIB)
+default:	prev_dllbuild cpl_config.h $(LIB)
 
 $(LIB):	$(OBJ)
 	lib /out:cpl.lib *.obj
 
 clean:
-	-del *.obj *.lib cpl_config.h
-
-cpl_config.h:	cpl_config.h.vc
-	copy cpl_config.h.vc cpl_config.h
+	-del *.obj *.lib cpl_config.h dllbuild.prev
+
+prev_dllbuild:
+	call <<prev_dllbuild.bat
+@SETLOCAL ENABLEEXTENSIONS
+IF NOT EXIST dllbuild.prev (ECHO $(DLLBUILD))> dllbuild.prev
+SET /P PREV_DLLBUILD=<dllbuild.prev
+IF NOT "%PREV_DLLBUILD%"=="$(DLLBUILD)" (ECHO $(DLLBUILD))> dllbuild.prev
+<<
```

Comment:
This "mechanism" updates a temporary `dllbuild.prev` file with the `$(DLLBUILD)` setting, but only when `dllbuild.prev` doesn't exist or doesn't match the current `$(DLLBUILD)` setting.

The upside to this is that repeated builds with the same setting won't cause `cpl_config.h` to be regenerated, which should prevent unnecessary recompilation.

Reply:
could you incorporate that as a comment in the file ?

Reply:
Sure thing!